core/neomutt.h

```diff
@@ -45,6 +45,7 @@ struct NeoMutt
   struct ConfigSubset *sub;      ///< Inherited config items
   struct AccountList accounts;   ///< List of all Accounts
   locale_t time_c_locale;        ///< Current locale but LC_TIME=C
+  mode_t userDefaultMode;        ///< User's default file writing mode (inferred from umask)
```

Comment:
Mind if we rename this to a C-style: `user_default_mode`

Reply:
> Mind if we rename this to a C-style: `user_default_mode`

Done!